packages/cli/src/commands/deploy/latest.js

```diff
@@ -295,7 +287,7 @@ export default async function main(
         output,
         'Which scope do you want to deploy to?',
         client,
-        ctx.config.currentTeam,
+        client.config.currentTeam,
```

Comment:
`selectOrg()` already has a client so can we remove `currentTeam` parameter here?